### PR TITLE
Change url structure

### PR DESCRIPTION
**Pull request by kevGalway**

### CODE REVIEW

File: _posts/2014-03-13-living-with-salespeople.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ---
+redirect_from:
```
Comment (kevGalway) [inline _posts/2014-03-13-living-with-salespeople.md:2]: All files below this point are just the redirects from old url to new url as per jekyll-redirect-from gem.

File: _config.yml [hunk 2]
```diff
@@ -34,6 +34,7 @@ plugins_dir:
 plugins:
   - jekyll-paginate
   - jekyll-sitemap
+  - jekyll-redirect-from
 
 paginate:               6
-permalink:              pretty
+permalink:              /:title/
```
Comment (kevGalway) [inline _config.yml:40]: Main change

File: _posts/2014-06-14-so-many-integrations.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
 ---
+redirect_from:
+  - /2014/06/16/So-many-Integrations,-such-little-time/
 layout: post
 title: "So many Integrations, such little time"
 slug: "So many Integrations, such little time"
```
Comment (anastasia-op) [inline _posts/2014-06-14-so-many-integrations.md:6]: That's a weird slug. Can we change it to `developing-crm-integrations`?

File: _posts/2014-07-21-api-tools-updated.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
 ---
+redirect_from:
+  - /2014/07/21/api-tools-updated/
 layout: post
 title: "API Tools Updated"
 category: blog
```
Comment (anastasia-op) [inline _posts/2014-07-21-api-tools-updated.md:6]: I don't see here the slug?

File: _posts/2016-11-18-remote-working-get-out-of-the-house.md [hunk 5]
```diff
@@ -1,4 +1,6 @@
 ---
+redirect_from:
+  - /2016/11/18/remote-working-Get-out-of-the-house!/
 layout: post
 title: "Remote Working - Get out of the house!"
 slug: "remote-working - Get out of the house!"
```
Comment (anastasia-op) [inline _posts/2016-11-18-remote-working-get-out-of-the-house.md:6]: Can we change the slug to `developers-working-remote`

File: _posts/2016-12-02-working_with_timezones_swift.md [hunk 6]
```diff
@@ -1,4 +1,6 @@
 ---
+redirect_from:
+  - /2016/12/02/working_with_timezones_swift/
 layout: post
 title: "Working with timezones (Swift)"
 slug: "working_with_timezones_swift"
```
Comment (anastasia-op) [inline _posts/2016-12-02-working_with_timezones_swift.md:6]: change the slub to `working-with-timezones-swift`

**Review by anastasia-op — COMMENTED** [review]
Left a few comments

**Pull request merged**